Utilities/bootstrap

```diff
@@ -287,6 +301,10 @@ class Target(object):
         common_args = ["-fobjc-arc", "-fmodule-name=%s" % self.module_name()]
         if platform.system() == 'Darwin':
             common_args.extend(["-arch", "x86_64", "-mmacosx-version-min=10.10"])
```

Comment:
Use: `-target x86_64-apple-macos10.10`

Reply:
Awesome, thats what I wanted to hear!  I was just afraid of changing it.